content/pages/api-versioning-strategy/index.md

```diff
@@ -0,0 +1,231 @@
+| Deprecation                                              | No              |
+| Retirement / End of Life                                 | Yes             |
+| Rename/reshape of a field, data type, or string constant | Yes             |
+| More restrictive change to field validations             | Yes             |
+| Less restrictive change to field validations             | No              |
+| Change to the HTTP status code returned by an operation  | Yes             |
+| Removal of a declared property                           | Yes             |
+| Removal or rename of APIs or API parameters              | Yes             |
+| Addition of a required request header                    | Yes             |
+| Adding more error codes                                  | No              |
+| Removing or modifying error codes                        | Yes             |
+| Addition of a member to an enumeration                   | Yes             |
+| Changing existing error response codes                   | Yes             |
+
+We use [oasdiff](https://github.com/Tufin/oasdiff/blob/main/BREAKING-CHANGES-EXAMPLES.md) tool to detect most of the possible breaking changes.
+
+## Support policy and deprecation information
+
+As new versions of the Box APIs and Box SDKs are released, earlier versions will be retired. Box declares a version as deprecated at least 24 months in advance of retiring it. Similarly, for individual APIs that are generally available (GA), Box declares an API as deprecated at least 24 months in advance of removing it from the GA version.
+
+When we increment the major version of the API (for example, from `2024.0` to `2025.0`), we're announcing that the current version (in this example, `2024.0`) is immediately deprecated and we'll no longer support it 24 months after the announcement. We might make exceptions to this policy for service security or health reliability issues.
+
+When an API is marked as deprecated, we strongly recommend that you migrate to the latest version as soon as possible. In some cases, we'll announce that new applications will have to start using the new APIs a short time after the original APIs are deprecated.
+
+When customer calls deprecated API endpoint, the response will contain a header:
+
+```sh
+Deprecation: date="Fri, 11 Nov 2018 23:59:59 GMT"
+Box-API-Deprecated-Reason: https://developer.box.com/reference/deprecated
+```
+
+The date tells clients when this version was marked as deprecated.
+
+## Versioning considerations
+
+When building your request, consider that:
+
+- If you do not specify a version, the service returns a default version that may not be the latest one.
+
+  - When the version header is missing, the default resource version is determined by the version in the URL.
+  - When the version header is specified, but the requested version is unavailable, then Box falls forward and responds to your request with the same behavior as the oldest supported stable version.
```

Comment:
I missed to update this part. It should be as following "When the Version header is specified, but the requested version is unavailable, then the response will return an HTTP error code 404 - Not Found."